.nvmrc

```diff
@@ -1 +1 @@
-v14
+v14.15.4
```

Comment:
This is the version we use in our CI scripts. Using something lower than 14.1 will fail when building the app due to https://github.com/maxogden/extract-zip/issues/94